noxfile.py

```diff
@@ -112,10 +135,24 @@ def _setup_template_environment(session: nox.Session) -> None:
 
 @nox.session(python="3.7")
 def install_bundled_libs(session):
+    # Install debugpy by url and platform
     """Installs the libraries that will be bundled with the extension."""
     session.install("wheel")
     _install_bundle(session)
 
+    target = os.environ.get("VSCETARGET", "")
+    print("target:", target)
+    if "darwin" in target:
+        download_url(f"{os.getcwd()}/bundled/libs", debugpy_urls["macOS"])
+    elif "win32-ia32" == target:
+        download_url(f"{os.getcwd()}/bundled/libs", debugpy_urls["win32"])
+    elif "win32-x64" == target:
+        download_url(f"{os.getcwd()}/bundled/libs", debugpy_urls["win64"])
+    elif "linux-x64" == target:
+        download_url(f"{os.getcwd()}/bundled/libs", debugpy_urls["linux"])
+    else:
+        download_url(f"{os.getcwd()}/bundled/libs", debugpy_urls["any"])
```

Comment:
Don't hard-code paths. You should use `pathlib.Path`:

```suggestion
        download_url(pathlib.Path.cwd() / "bundled" / "libs", debugpy_urls["any"])
```